content/blog/gdpr-cheat-sheet.en.md

```diff
@@ -0,0 +1,139 @@
+
+Our [Data Protection Supervisory Authority Finder]({{< ref "supervisory-authorities" >}}#finder) will help you find the supervisory authority responsible for you.
+
+### Personal Data
+
+The concept of personal data is the crux of the GDPR. Almost all of the regulations refer to it. As soon as information can be assigned to a specific person, it is considered personal data. A name is not necessarily required for the assignment. For example, an IP address, a user name, a cookie, or an identification number would suffice.
+
+The information that the IP address 31.41.59.26 called our [generator]({{< ref "generator" >}}) on 25 May 2018 would therefore be considered personal data. On the other hand, the information that on the same day in a bakery a third of the customers paid with cash is not.
+
+### Profiling
+
+Profiling is a form of automated processing where the [controller](#data-controller) uses your [personal data](#personal-data) to automatically evaluate certain personal aspects relating to you without any human intervention. It is often a matter of analysing and predicting aspects such as the following:
+
+* Work performance
+* Economic situation
+* Health
+* Personal preferences
+* Interests
+* Reliability
+* Behaviour
+* Location or movements
+
+A classic example is the mobile phone or credit card credit check that you need to pass before getting a contract. Many companies will forward your data to credit agencies such as [Schufa]({{< ref "schufa" >}}), [Creditreform Boniversum]({{< ref "creditreform-boniversum" >}}) or [infoscore]({{< ref "infoscore" >}}). These agencies will then determine how likely you are to fulfil your contractual obligations based on information such as your previous payment behaviour or the number of bank accounts that you have.
+
+### Rights
+
+The GDPR grants you extensive rights with regards to your [personal data](#personal-data). This starts with the right to information: If a [controller](#data-controller) wants to [process](#processing) your data, they must inform you about it and tell you how you can contact them if you have any questions.
+
+Even after the processing of your data has started, you still have numerous other rights. For example, you can ask what personal data is being processed and request a copy of it. You can also request for incorrect data to be corrected and under certain circumstances, even have your data deleted.
+
+You will find a comprehensive overview and a detailed explanation on this subject in our [article on your GDPR rights]({{< ref "your-gdpr-rights" >}}).
+
+### Data Controller
```

Comment:
Not the first instance where this is relevant but here's where I noticed it: In the German post, I sorted the heading alphabetically. We don't need to do that here, though. I'll leave the decision to you.